cli/README.md

```diff
@@ -1,100 +1,100 @@
-# Introducing the xi-iot Command Line Interface (CLI)
+# Introducing the kpsctl Command Line Interface (CLI)
 
-**Note**: Nutanix is actively developing the xi-iot CLI. As a result, the software published here is release candidate (RC) quality until further notice. 
+**Note**: Nutanix is actively developing the kpsctl CLI. As a result, the software published here is release candidate (RC) quality until further notice. 
 Use this software at your own risk and do not use it in a production environment or deployment. The latest release candidate is shown below.
 
-The xi-iot CLI is a command line tool that helps you manage your Xi IoT resources from your terminal or shell. With minimal 
-configuration, you can start using Xi IoT functionality equal to that provided by the browser-based Xi IoT Management 
+The kpsctl CLI is a command line tool that helps you manage your Karbon Platform Services for IoT resources from your terminal or shell. With minimal 
+configuration, you can start using Karbon Platform Services for IoT functionality equal to that provided by the browser-based Karbon Platform Services for IoT Management 
 Console from the command prompt in your favorite terminal program.
 
 ## Shell Support
 **Linux and macOS** You can use common shell programs such as bash and zsh.
 Note that autocompletion for zsh is experimental in this release.
 
-# Installing xi-iot CLI
+# Installing kpsctl CLI
 ## MacOS
-Download the xi-iot CLI for MacOS:
+Download the kpsctl CLI for MacOS:
 
 **64-bit**
-[xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz](https://xi-iot-cli.s3-us-west-2.amazonaws.com/xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz)
+[kpsctl-v1.0.0-rc5-darwin_amd64.tar.gz](https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc5-darwin_amd64.tar.gz)
 
 **32-bit**
-[xi-iot-v1.0.0-rc5-darwin_386.tar.gz](https://xi-iot-cli.s3-us-west-2.amazonaws.com/xi-iot-v1.0.0-rc5-darwin_386.tar.gz)
+[kpsctl-v1.0.0-rc5-darwin_386.tar.gz](https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc5-darwin_386.tar.gz)
 
 Optionally, from your terminal or shell, use `wget`.
 ```
-$ wget https://xi-iot-cli.s3-us-west-2.amazonaws.com/xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz
-$ mkdir xi-iot && tar zxvf xi-iot-v1.0.0-rc5-darwin_amd64.tar.gz -C xi-iot
-$ sudo xi-iot/install [*optional_path*]
+$ wget https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc5-darwin_amd64.tar.gz

```

Comment:
These links are giving 403 forbidden, assume we need to change permissions

Reply:
No bucket permissions are fine, but I need to change version in link to rc7: wget https://kpsctl-cli.s3-us-west-2.amazonaws.com/kpsctl-v1.0.0-rc7-darwin_amd64.tar.gz

Reply:
updated to rc8